views/place-form.hbs

```diff
@@ -7,39 +7,77 @@
       <h1>{{localText.placesAdd}}</h1>
       <input type="hidden" name="_method" value="post">
   {{/if}}
+  <span> Details below are required in at least one of Arabic and English.</span>
 
-  <label>{{localText.placeName}}
-    {{#if place.local.name}}
-      <input type="text" name="name" required value="{{place.local.name}}">
-    {{else}}
-      <input type="text" name="name" required>
-    {{/if}}
-  </label>
+  <fieldset name="en" class="langFieldset">
+    <legend>{{localText.english}}</legend>
+    <label>{{localText.placeName}}
+      {{#if place.local.name}}
+      <input type="text" name="name_en" required value="{{place.local.name}}">
```

Comment:
All these inputs need indenting inside the if statements :v:

Reply:
oh come on

Reply:
lololololol get to work